### PR TITLE
Update axios to 1.6.2 to remove vulnerabilities

### PR DESCRIPTION
**Pull request by kyrielia**
## Summary

See: https://github.com/contentful/contentful-migration/issues/1267

## Description

Upgrade axios to latest available version

## Motivation and Context

Using contentful-migration introduces a vulnerable transitive dependency into any user that uses this library. This fix will mitigate this.

### CODE REVIEW

**Comment by kyrielia** [comment]
I've made a similar PR in contentful-management:
https://github.com/contentful/contentful-management.js/pull/2055

Not 100% sure, but I think this PR might have to wait until the fix in contentful-management is merged in? It looks like contentful-migration uses management:
https://github.com/contentful/contentful-migration/pull/1268/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L60

**Review by t-col — APPROVED** [review]

**Comment by t-col** [comment]
thanks for your contribution @kyrielia ! you're right that the axios 1.5.1 dependency won't be entirely removed until we merge your other PR in contentful-management.js and then update the version of that dependency here. But we'll do that separately.

**Pull request merged**
